content/blog/automating-operations-on-dscc-using-ansible-playbooks.md

```diff
@@ -0,0 +1,440 @@
+---
+title: Automating operations on Data Services Cloud Console using Ansible Playbooks
+date: 2023-03-29T14:00:40.866Z
+author: Anusha Y and Sijeesh Kattumunda
+authorimage: /img/Avatar1.svg
+disable: false
+---
+Automation is one of the top trends in technology and the pace of automation is accelerating with more companies opting for developing fully automated systems. Automation reduces time, effort, cost, and manual errors while increasing efficiency and productivity. Gone are those days when many complex coding skills were required to implement automation. Now, there are many low-code tools available in the market, like Ansible, that make automation easier.
+
+In this blog post, I am excited to be able to introduce the Ansible playbooks for Data Services Cloud Console and show you how to use them. Along with the [Python SDK](https://github.com/HewlettPackard/greenlake-data-services-python) for Data Services Cloud Console, these playbooks should help you in your efforts to automate HPE GreenLake Data Services through an infrastructure-as-code approach.
+
+Ansible is an open-source IT automation tool that automates provisioning, configuration management, application deployment, and many other IT processes. 
+Two main features that make Ansible the best choice for automation are:
+
+* Ansible does not require any programming. 
+* Idempotence is offered as a built-in feature of many of the Ansible modules. This means the result of performing a task once is the same as performing it multiple times without any intervening actions.
+
+**Why do we need Ansible playbooks for Data Services Cloud Console?**
+
+Ansible helps the users/admins automate the deployment of resources and applications without the manual overhead of creating everything from scratch. These playbooks can be configured with conditions, variables, and tasks. Currently, simple playbooks, like performing CRUD operations on the resources, are available. These playbooks can be considered basic building blocks and can be reused to build simple-to-complex use cases.
+
+**[Ansible Modules for  Data Services Cloud Console](https://github.com/HewlettPackard/greenlake-data-services-ansible):**
+
+The following Ansible modules are currently available for Data Services Cloud Console. You can use the samples given or customize it - 
```

Comment:
In this instance, you can leave it in the shortened form of "Data Services Cloud Console" because it's not in a header or title and you've already mentioned it fully in the body text above earlier.

Reply:
I did have the shortened version but Denis pointed out that we cannot use it, as mentioned here https://confluence.eng.nimblestorage.com/display/TPT/Product+nam